app/src/components/calendar-main.tsx

```diff
@@ -26,9 +25,8 @@ interface CalendarEvent {
   description: string;
 }
 
-// Function to format the date
 const formatDate = (dateString: string): string => {
-  const date = new Date(dateString + 'T00:00:00Z'); // Ensure it's treated as UTC
+  const date = new Date(dateString + 'T00:00:00');
```

Comment:
This fixes the time zone offset that was previously occurring where you would select November 20 and it would say November 19